package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "fuse.js": "^6.6.2",
     "mixpanel-browser": "^2.47.0",
+    "simple-icons": "^13.3.0",
```

Comment:
does this one replace the bootstrap one?
`"svelte-bootstrap-icons": "^3.0.0",`

Reply:
Simple Icons is only for company logos, so it doesn't have the chevron needed for the back to top button 😞 .  That chevron will still need to come from the `svelte-bootstrap-icons` package.